internal/databases/sqlserver/blob/server_test.go

```diff
@@ -0,0 +1,47 @@
+package blob
+
+/*
```

Comment:
Why commented?

Reply:
Sorry, an unnecessary file was added to commit.  Removed it.